Tools/build-translation.js

```diff
@@ -56,13 +56,26 @@ function buildLocale(inputFile, outputFile) {
 	saveToFile(outputFile, translation);
 }
 
-async function removePoHeaderDate(filePath) {
-	// Note: on macOS this will fail because it needs to be 'sed -i ""'
-	// Solution would be to install gsed, detect it here, and use it in place of sed in macOS
-	// https://stackoverflow.com/questions/30003570/how-to-use-gnu-sed-on-mac-os-x#34815955
+function returnPath(file) {
```

Comment:
It's implied that it returns something. So maybe call it "executablePath(file)" to tell what it returns?

Reply:
Hmm, I read it like this: returnPath for file.

But if you want to change it, how about `pathOfExecutable`?

Reply:
Right I didn't realise. pathOfExecutable is good too 👍

Reply:
I've already used your first suggestion a few hours ago.